app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -60,6 +73,17 @@ def generate_pdf(_form526_submission, _assessed_data)
       raise "Method `generate_pdf` should be overriden by the subclass #{self.class}"
     end
 
+    # Override this method to add to form526_submission.form_json['rrd_med_stats']
+    def med_stats_hash(_form526_submission, _assessed_data); end
+
+    # @param assessed_data [Hash] results from assess_data
+    def add_medical_stats(form526_submission, assessed_data)
+      med_stats_hash = med_stats_hash(form526_submission, assessed_data)
+      return if med_stats_hash.blank?
+
+      self.class.add_medical_stats_hash(form526_submission, med_stats_hash)
+    end
```

Comment:
Do we know enough about other types of RRD claims to move towards generalizability?

Reply:
Not yet, which is why the only non-empty implementation of `med_stats_hash` is in `Form526HypertensionJob`.